assets/src/edit-story/components/panels/design/pageBackground/pageBackground.js

```diff
@@ -0,0 +1,175 @@
+`;
+
+const MediaWrapper = styled.div`
+  height: 24px;
+  width: 24px;
+  border-radius: 4px;
+  overflow: hidden;
+  margin: 6px;
+  img {
+    object-fit: cover;
+    min-height: 24px;
+  }
+`;
+
+const RemoveButton = styled(Button)`
+  margin: 1px;
+  align-self: center;
+`;
+
+const Text = styled(DefaultText)`
+  align-self: center;
+  width: 55px;
+`;
+
+function PageBackgroundPanel({ selectedElements, pushUpdate }) {
+  const {
+    state: { currentPage },
+    actions: { updateCurrentPageProperties },
+  } = useStory();
+
+  const updateBackgroundColor = useCallback(
+    (value) => {
+      updateCurrentPageProperties({ properties: { backgroundColor: value } });
+    },
+    [updateCurrentPageProperties]
+  );
+
+  const { setBackgroundElement } = useStory((state) => ({
+    setBackgroundElement: state.actions.setBackgroundElement,
+  }));
+
+  const removeAsBackground = useCallback(() => {
+    pushUpdate(
+      {
+        isBackground: false,
+        opacity: 100,
+        overlay: null,
+      },
+      true
+    );
+    setBackgroundElement({ elementId: null });
+  }, [pushUpdate, setBackgroundElement]);
+
+  const backgroundEl = selectedElements[0];
+  if (!backgroundEl || !backgroundEl.isBackground) {
+    return null;
+  }
+  const isDefaultBackground = backgroundEl.isDefaultBackground;
+  const isMedia = backgroundEl.isBackground && !isDefaultBackground;
+
+  const { backgroundColor } = currentPage;
+  const { LayerContent } = getDefinitionForType(backgroundEl.type);
+
+  // Background can only have one selected element.
+  const flip = selectedElements[0]?.flip || DEFAULT_FLIP;
+
+  return (
+    <SimplePanel
+      name="pageBackground"
+      title={__('Page background', 'web-stories')}
+      isPersistable={false}
+    >
+      {isDefaultBackground && (
+        <Row>
+          <Color
+            hasGradient
+            value={backgroundColor}
+            onChange={updateBackgroundColor}
+            label={__('Background color', 'web-stories')}
+            colorPickerActions={getColorPickerActions}
+            hasOpacity={false}
+          />
+        </Row>
+      )}
+      {isMedia && (
+        <Row expand={false}>
+          <SelectedMedia>
+            <MediaWrapper>
+              <LayerContent element={backgroundEl} height={'100%'} />
+            </MediaWrapper>
+            <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
+              {__('Media', 'web-stories')}
+            </Text>
+            <RemoveButton
+              aria-label={__('Detach background', 'web-stories')}
+              onClick={removeAsBackground}
+              size={BUTTON_SIZES.SMALL}
+              variant={BUTTON_VARIANTS.SQUARE}
+              type={BUTTON_TYPES.TERTIARY}
+            >
+              <Icons.Cross />
+            </RemoveButton>
+          </SelectedMedia>
+          <FlipControls
+            onChange={(value) => pushUpdate({ flip: value }, true)}
+            value={flip}
+          />
```

Comment:
There's a lot of tests failing now due to either the detach button being renamed or the flip controls not working as expected. I think it's fairly easy to fix, but some of it is spread over many files.